packages/strapi-plugin-content-type-builder/admin/src/components/AttributeRow/index.js

```diff
@@ -0,0 +1,79 @@
+/**
+*
+* AttributeRow
+*
+*/
+
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { capitalize } from 'lodash';
+import Boolean from '../../assets/images/icon_boolean.svg';
+import Date from '../../assets/images/icon_date.svg';
+import Image from '../../assets/images/icon_image.svg';
+import Json from '../../assets/images/icon_json.svg';
+import Relation from '../../assets/images/icon_relation.svg';
+import String from '../../assets/images/icon_string.svg';
+import Text from '../../assets/images/icon_text.svg';
+import styles from './styles.scss';
+
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+class AttributeRow extends React.Component { // eslint-disable-line react/prefer-stateless-function
+  constructor(props) {
+    super(props);
+    this.asset = {
+      'boolean': Boolean,
+      'date': Date,
```

Comment:
It will be better to not use a reserved word. For example, you could add a prefix `icon_` instead